fix: clarify Dynamic API calls in wrong context (#62143)
vercel/next.js: errors/next-headers-wrong-context.mdx

---
title: next/headers methods were called outside request/render scope
---

#### Why This Error Occurred

> Either `headers()` or `cookies()` were invokned at place where no request information was available (eg.: Global scope, outside the React lifecycle).

Reviewer: react lifecycle is not required, such as app route handler

```suggestion
Either `headers()` or `cookies()` were invoked somewhere outside of the request scope (e.g. Global scope).
```

Author: See https://github.com/vercel/next.js/pull/62143#discussion_r1493464969
